content/v1/features/powerflex.md

```diff
@@ -497,6 +497,26 @@ or you could edit logConfig.yaml, and use this command:
 `kubectl apply -f logConfig.yaml`  
 
 and then make the necessary adjustments for LOG_LEVEL and LOG_FORMAT.  
-If LOG_LEVEL or LOG_FORMAT are set to options outside of what is supported, the driver will use the default values of "info" and "text" . 
+If LOG_LEVEL or LOG_FORMAT are set to options outside of what is supported, the driver will use the default values of "info" and "text" .   
 
+## Volume Health Monitoring 
 
+Starting in version 2.1, CSI Driver for PowerFlex now supports volume health monitoring. This allows Kubernetes to report on the condition of the underlying volumes via events when a volume condition is abnormal. For example, if a volume were to be deleted from the array, or unmounted outside of Kubernetes, Kubernetes will now report these abnormal conditions as events.  
+
+To accomplish this, the driver utilizes the sidecar, external-health-monitor. When driver detects a volume condition is abnormal, the sidecar will report an event to the corresponding PVC. For example, in this event from `kubectl describe pvc -n <ns>` we can see that the underlying volume was deleted from the PowerFlex array:
```

Comment:
I would rephrase the first sentence as "To accomplish this, the driver utilizes the external-health-monitor sidecar"

Reply:
done